ViteMaDose/Views/CentresList/Cells/CentreCell.swift

```diff
@@ -137,6 +142,12 @@ final class CentreCell: UITableViewCell {
         vaccineTypesLabel.text = viewData.vaccineTypesText
         vaccineTypesLabel.font = Constant.labelPrimaryFont
         vaccineTypesLabel.textColor = Constant.labelPrimaryColor
+
+        centerTypeContainer.isHidden = viewData.centerTypeText == nil
+        centerTypeLabel.text = viewData.centerTypeText
+        centerTypeLabel.font = Constant.labelPrimaryFont
+        centerTypeLabel.textColor = Constant.labelPrimaryColor
```

Comment:
Worth adding this maybe
```suggestion
        centerTypeLabel.textColor = Constant.labelPrimaryColor
        centerTypeLabel.numberOfLines = .zero
```

Reply:
Ok